Site refactor - phase one (#402)
solid/solidproject.org: pages/home.md

>   <div class="hero-body">

Reviewer: With all due respect, but I think this is quite a setback for the homepage. Horizontally centring of longer stretches of text makes them very hard to read, and it just does not feel as a cohesive whole - the first impression is of a somewhat amateurish website, IMO. I'd prefer to try to fit the new content into the existing design Lisa made earlier, and possibly to ask her (or someone else) to update the design based on the new content, rather than trying to cobble something together quickly that looks the part.

(Also sorry that I'm being a bit rough here; if we didn't have the time pressure I could provide a more constructive path forward, but in it's current state, I don't think it's actually an improvement over what we have.)

Reviewer: Also, aren't we writing "Pod" with a capital P? :)

Author: Definitely not meant to be the long-term design. Already have a request in for design help, but wanted to get the content updated first, and didn't have the corresponding graphics available for it. Will see if something can be done today.

Reviewer: So how about replacing the text that currently accompany the illustrations by context you've added, and simply removing the buttons? Even if the illustrations do not match the text, it'd still look a whole lot more professional?

Author: I'm talking through this with @tony-morelli now - but i agree and i think we can settle it today. Main question is whether or not to push this in the meantime and then update in an iteration tonight or tomorrow.

    <div class="container">
      <h1 class="title has-text-centered is-size-3 is-size-2-tablet is-size-1-desktop">

>         Solid

Reviewer 2: Here, the idea was that `Solid` is already present in the top left hand corner of the page, so the space could be used for a tagline, rather than repeating `Solid`. Moreover, this linked to the description of the intent of Solid by Tim, which sends a powerful message.

Author: We're going to add a vision statement back in under the roadmap section, in conjunction with some history copy that tells the story well. We could look at swapping the name with the logo here instead.

Author: @NSeydoux replaced the "Solid" text with the logo. Looks nicer this way ✅

      </h1>
      <p class="subtitle has-text-centered is-size-4 is-size-3-tablet is-size-2-desktop">
      Using Web standards to
      let people control their data, and choose the
      applications and services to use with it.
      </p>
      <div class="buttons is-centered">
        <!--
        <a href="{{site.baseurl}}/about/">
          <button class="button is-primary">About Solid</button>
        </a>
        -->

>         <a href="{{site.baseurl}}/users/">
>           <button class="button is-primary">Get a Pod</button>        
>         </a>
>         <a href="{{site.baseurl}}/developers/tutorials/getting-started">
>           <button class="button is-primary">Developer Tutorial</button>
>         </a>

Reviewer: A couple of changes:

- Screen readers won't be able to make sense of buttons inside anchors, probably - `<button>`s removed.
- Increased the button sizes to fit the visual hierarchy.
- Only made a single button the primary call to action to clearly guide the eye.

```suggestion
        <a href="{{site.baseurl}}/users/" class="button is-primary is-large">
          Get a Pod
        </a>
        <a href="{{site.baseurl}}/developers/tutorials/getting-started" class="button is-info is-outlined is-large">
          Developer Tutorial
        </a>
```

      </div>
    </div>
  </div>
</section>

<div id="landing-content">

  <section id="overview" class="section">

    <div class="container">

        <p class="subtitle has-text-centered is-3">
          Solid lets people store their data in decentralized data stores 

>           called <b>pods</b>.

Reviewer 2: ```suggestion
          called <b>Pods</b>.
```

        </p> 

        <div class="columns">
          <div class="column is-half-desktop is-offset-one-quarter-desktop">
          
            <p class="content has-text-centered is-large">
              Pods are like secure personal web servers for data.

>               All data in a pod is accessible
>               accesible via the 

Reviewer 2: ```suggestion
              All data in a pod is
              accesible via the 
```

>               <a href="https://solid.github.io/specification/">Solid Protocol</a>.

Reviewer: Do we need _another_ word for this? I don't think I've ever seen "Protocol" used yet...

Author: This is something that was agreed upon by Editors and named by Tim. The main client/server specification is going to be named in the specification as the "Solid Protocol".

Reviewer: That's good to know. So basically, almost everywhere we've been using "Solid standard(s)" and "Solid specification(s)" before, we should be using "Solid Protocol" from now on?

Author: Solid Specification is still good as an umbrella term for all pertinent specifications underneath. For example, Solid Protocol is the core client/server specification, but wouldn't include the spec for application interoperability. Solid Specification would encompass them both.

Reviewer: Right, thanks. I think that doesn't apply for most of our uses of it in e.g. our documentation, so I'll feed that back to the relevant people.

              When data is stored in someone's 
              pod, they control who and what can access it.
            </p>
            
            <p class="content has-text-centered is-large">
            Solid is led by the inventor of the Web, 
            <b><a href="{{site.baseurl}}/team/">Sir Tim Berners-Lee</a></b>, 

>             to help realize his vision 

Reviewer: Just realised: we've been using British English throughout the site so far. I have no idea how that's perceived by native speakers, so I'm just flagging it.

Reviewer 3: Good point. I'd say it's more important to be consistent, so I'd edit this to be in line with whatever the rest of the site currently uses (and then open an issue to get **_formal agreement_** (if there isn't already) on UK vs US English overall).

Author: Agree with @pmcb55. For now I've corrected that to use british english for consistency.

            for its future.</p>
            
          </div>
        </div>
        
    </div>
  </section>
  
  <section id="landing-cards">

    <div class="container">
    
      <div class="columns">
  
        <div class="column">

          <div class="card">
            <div class="card-body has-text-centered section">
              <figure class="image is-64x64">
                <img
                  src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/solid/download.svg"
                  alt="[]"
                />
              </figure>
              <p class="title is-size-4 is-uppercase">
                <strong>Store anything</strong>
              </p>
              <hr/>
              <p class="content is-size-5">

>                 Any kind of data can be stored in a Solid pod, including regular files that you might store in a Google Drive or Dropbox folder, but it is the ability to store Linked Data that makes Solid special.

Reviewer: Food for thought: while it may not seem that way when surrounded by Solid people, "linked data" and the eternal promise of interoperability and data that's understandable by both humans and computers would be an instant turn-off for me as someone not coming from the RDF world (but having been there when it was overhyped 15 years ago). So maybe we shouldn't put it this front-and-center without speaking more to why it's useful for Solid to build on that and why-it's-different-this-time-no-really.

Author: We've been criticized in the last site design for not explaining what Solid actually is. The vast majority of people will not have any past history with or knowledge of linked data. If we explain why this time is different up front, we'll confuse them all.

Reviewer 3: Interesting perspective @Vinnl - but I'd say there is **_nothing_** "different-this-time" at all - i.e. we're still using the same RDF (basically, v1.1 did clean up some of the [crufty stuff](https://www.urbandictionary.com/define.php?term=cruft)) that was around 15 years ago. So in fact we've nothing really to "speak more to" here. What **_has_** changed though (slowly, but increasingly) is people's realization that RDF really might actually be a-bit-better-and-actually-usable-in-practice, at least more so than all the previous attempts at distributed interop (e.g. SOAP, DCOM, XML-RPC, CORBA, etc.). Probably best articulated by Dan Brickley + Libby Miller (this quote is definitely gonna become a major meme :) - https://book.validatingrdf.com/bookHtml005.html). To your original point though, maybe the text here could allude to that quote perhaps (or just repeat the first three short sentences).

                <strong>Use Interoperable Data Standards</strong>
              </p>
              <hr/>
              <p class="content is-size-5">
                Linked Data gives Solid 
                a common way to describe things and 
                how they relate to each other, in a way that other people and 
                machines can understand. This means the data stored by Solid
                is portable and completely interoperable.
              </p>
            </div>
          </div>
        
        </div>
        <div class="column">    
        
          <div class="card">
            <div class="card-body has-text-centered section">
              <figure class="image is-64x64">
                <img
                  src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/solid/share-square.svg"
                  alt="[]"
                />
              </figure>
              <p class="title is-size-4 is-uppercase">
                <strong>Share it Safely</strong>
              </p>
              <hr/>
              <p class="content is-size-5">
                Anyone or anything that accesses data in a Solid pod uses a 

>                 unique ID, authenticated by a decentralized OpenID Connect 

Reviewer: ```suggestion
                unique ID, authenticated by the decentralized OpenID Connect 
```

Author: Used 'a' because Solid-OIDC is not _the_ de-facto decentralized OIDC protocol (at least not yet 🙂 )

Reviewer: Oh really? I wasn't aware - maybe that needs clarification here as well then, as I thought there was just one OIDC protocol? Is more like an extension of OIDC then?

Author: Yes it's been extended - see https://solid.github.io/authentication-panel/solid-oidc/

Reviewer: Would this help clarify?

```suggestion
                unique ID, authenticated by an extension of the decentralized OpenID Connect 
```

Author: should be "a decentralized extension of the OpenID Connect..."

Reviewer: Haha, shows how much the clarification is needed I guess :P 

```suggestion
                unique ID, authenticated by a decentralized extension of OpenID Connect 
```
